src/dbnode/storage/database.go

```diff
@@ -340,6 +358,32 @@ func (d *db) addNamespacesWithLock(namespaces []namespace.Metadata) error {
 	return nil
 }
 
+func (d *db) updateNamespaceSchemasWithLock(schemaUpdates []namespace.Metadata) error {
+	for _, n := range schemaUpdates {
+		// ensure namespace exists
+		existing, ok := d.namespaces.Get(n.ID())
+		if !ok { // should never happen
```

Comment:
Again, I know the existing file breaks this convention but we usually try and put a comment like this within the branch so:

```golang
if !ok {
    // Should never happen.
    ....
}
```

Reply:
ok